bundles/org.openhab.binding.zoneminder/src/main/resources/OH-INF/config/config.xml

```diff
@@ -0,0 +1,91 @@
+			<default>true</default>
+		</parameter>
+		<parameter name="discoveryInterval" type="integer" min="60" required="false" unit="s"
+			groupName="config-info">
+			<label>Monitor Discovery Interval</label>
+			<description>Specifies time in seconds in which the binding will attempt to discover monitors</description>
+			<default>300</default>
+		</parameter>
+		<parameter name="defaultAlarmDuration" type="integer" unit="s" required="false" groupName="config-info">
+			<label>Default Alarm Duration</label>
+			<description>Duration in seconds after which the alarm will be turned off</description>
+			<default>60</default>
+		</parameter>
+		<parameter name="defaultImageRefreshInterval" type="integer" required="false" groupName="config-info">
+			<label>Default Image Refresh Interval</label>
+			<description>Interval in seconds at which monitor image snapshot will be updated</description>
+		</parameter>
+		<parameter name="host" type="text" required="true" groupName="url-info">
+			<label>Server</label>
+			<description>ZoneMinder server name or IP address</description>
+			<context>network-address</context>
+		</parameter>
+		<parameter name="useSSL" type="boolean" required="true" groupName="url-info">
+			<label>Use https</label>
+			<description>Enables use of https for connection to ZoneMinder</description>
+			<default>false</default>
+		</parameter>
+		<parameter name="portNumber" type="integer" min="1" max="65535" required="false" groupName="url-info">
+			<label>Port Number</label>
+			<description>Port Number</description>
+		</parameter>
+		<parameter name="useDefaultUrlPath" type="boolean" required="false" groupName="url-info">
+			<label>Use Default URL Path</label>
+			<description>Use standard URL path (i.e. /zm)</description>
+			<default>true</default>
+		</parameter>
+		<parameter name="urlPath" type="text" required="false" groupName="url-info">
+			<label>URL Path</label>
+			<description>URL path (Used if ZoneMinder URL starts with something other than /zm)</description>
+		</parameter>
+		<parameter name="user" type="text" required="false" groupName="auth-info">
+			<label>User Name</label>
+			<description>User name (if authentication enabled in ZoneMinder)</description>
+		</parameter>
+		<parameter name="pass" type="text" required="false" groupName="auth-info">
+			<label>Password</label>
+			<description>Password (if authentication enabled in ZoneMinder)</description>
+			<context>password</context>
+		</parameter>
+	</config-description>
+
+	<config-description uri="thing-type:zoneminder:monitor">
+		<parameter name="monitorId" type="text" required="true">
+			<label>Monitor Id</label>
+			<description>Id of the monitor</description>
+		</parameter>
+		<parameter name="imageRefreshInterval" type="integer" unit="s" required="false">
+			<label>Image Refresh Interval</label>
+			<description>Interval in seconds with which monitor image is refreshed</description>
+		</parameter>
```

Comment:
If it isn't required what is the default value?
Also shouldn't this have a minimum value?

Reply:
No, the default is to not refresh the image, in which case you leave the field blank.

Yes, I'll add a minimum.